enhancements/machine-api/machine-api-usage-telemetry.md

```diff
@@ -0,0 +1,338 @@
+future planning and guidance. Proactive support speaks to information that we
+learn which will lead to better experiences for users by providing opportunity
+for preventitive intervention.
+
+**Total Remediation Events**
+
+Epics: Demonstration of Value, Proactive Support
+
+This metric will give us insight into how effectively the MHC is being used.
+A sustained high number of events on a particular cluster will enable us to
+deliver proactive support. A sudden spike across all clusters might give us
+early warning to a cloud provider’s data center outage, or a large network
+or similar event that is impacting many of our users.
+
+For example, since many clusters operate at a relatively small scale, it’s
+possible that losing an AZ in US-East-1 would be small enough event on their
+cluster to trigger MHC. Letting users know we’re seeing this across many
+clusters would assist in their own troubleshooting efforts and save them time.
+
+Having a total number of remediation events can demonstrate the value our
+platform is bringing to our users. It will show the operational burden we’ve
+decreased on teams that use OpenShift.
+
+**Remediation is Disabled or Short Circuited**
+
+Epics: Proactive Support
+
+This metric will give us insight into when a particular cluster has too many
+unhealthy nodes for the MHC to do its job. A user may be expecting the MHC is
+running as normal, cleaning up broken hosts, when in reality it has been silently disabled.
+Any sustained disabling of the MHC should potentially trigger a proactive
+support engagement. Similar to the _Total Remediation Events_, if any clusters
+experience this simultaneously, we can help provide any potential outage info
+to our users.
+
+**Number of Nodes Covered by the Machine Health Check**
+
+Epics: Engineering Insights, Proactive Support
+
+Users might think they are utilizing the MHC correctly only to find out that
+it has been misconfigured and not performing its role. If users have a large
+cluster with the MHC enabled, but not covering any nodes, this might be an
+opportunity to reach out and see if we can help with their configuration.
+Having this data will give us insight to drive the user experience of the MHC.
+
+**Failed Node Cleanups**
+
+Epics: Proactive Support
+
+This metric will help us identify users that are having problems with their
+infrastructure that might go unnoticed. We should trigger an alert if the
+autoscaler cleans up more than X hosts. This alert would be useful for users
+that might not realize they’re having these infrastructure problems.
+This is another case where signals across multiple clusters might indicate
+that a cloud or network area is having problems.
+
+For example, Our internal CI infra team had multiple MachineSets configured
+with the autoscaler.  One MachineSet would scale up fine, The other wouldn’t.
+The machines would fail, alarms would fire, the autoscaler would dispose of
+the nodes that failed to come online silently.
```

Comment:
This paragraph mixes casing on resource names, should keep them consistently camel cased (MachineSet, Machine, Node)

Reply:
i'll clean this up